docs/GettingStartedOnLinux.md

```diff
@@ -0,0 +1,217 @@
+---
+id: getting-started
+title: Getting Started on Linux
+layout: docs
+category: Quick Start
+permalink: docs/getting-started-on-linux.html
+next: android-setup
+---
+
+## Intro
+
+This guide is essentially a beginner-friendly version of the [Getting Started](/react-native/docs/getting-started.html) page for React Native on Linux.
+
+### Platform
+
+For the purposes of this guide, we assume that you're working on Ubuntu Linux 14.04 LTS.
+
+### Prerequisites
+
+Before following this guide, you should have installed the Android SDK and run a successful Java-based "Hello World" app for Android.
+
+Other than that, we should have you covered!
+
+Let's dig in.
+
+### Installing everything
+
+#### Install NodeJS
+
+The first thing you need to do is to install NodeJS, a popular Javascript implementation.
+
+Fire up a terminal and paste these commands in to install NodeJS from APT:
+
+* `sudo apt-get install -y build-essential`
+* `curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -`
+* `sudo apt-get install -y nodejs`
+
+#### Install NPM
```

Comment:
Starting on 14.04 it's no issue. I can remove this if need be. I added this to catch laggards.